opa-manager-operator/testing-yamls/sync.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
```

Comment:
Where is this file used?

Reply:
these are example files, maybe I should not have commited. Will remove from next commit